packages/ui/components/core/Footer.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Meta, StoryObj } from '@storybook/react'
+
+import { Footer } from './Footer'
+
+export default {
+	title: 'Design System/Footer',
+	component: Footer,
+	parameters: {
+		layout: 'fullscreen',
+		design: {
+			type: 'figma',
+			url: 'https://www.figma.com/file/gl8ppgnhpSq1Dr7Daohk55/Design-System-(2023)?t=cm6XAiXXhLOXJAQY-0',
+		},
+	},
```

Comment:
This will emulate the grid setup from the page level.

```suggestion
			layout: 'fullscreen',
	},
	decorators: [StorybookGrid],
```